grails-app/views/notify/dulFormLink.gsp

```diff
@@ -0,0 +1,9 @@
+<p>
+Hello,
+This email is a request from your collaborator at the Broad Institute to have your IRB or Ethics Committee of record
+complete the linked <b><i>Data Use Letter</i></b> form by interpreting the consent form for this study.
+Could you please share <a href="${details}">this link</a> and/or email with a member of your IRB or Ethics Committee so they can proceed with filling out and submitting the Data Use Letter form?
+If you have any questions, please do not hesitate to email your collaborator (cc'd) or ORSP@broadinstitute.org.
+Thank you,
+Sent on behalf of Broad Institute's Office of Research Subject Protection (ORSP)
+</p>
```

Comment:
nit: without breaks, this won't format in the way it looks here. Just want to make sure it really looks the way you want it to. There is a mail group that all email goes through that you can get access to (will post in slack) so you can see what it looks like for real.